src/plugins/data/public/ui/filter_bar/filter_item.tsx

```diff
@@ -349,32 +350,44 @@ export function FilterItem(props: FilterItemProps) {
     return null;
   }
 
-  const badge = (
-    <FilterView
-      filter={filter}
-      valueLabel={valueLabelConfig.title}
-      filterLabelStatus={valueLabelConfig.status}
-      errorMessage={valueLabelConfig.message}
-      className={getClasses(filter.meta.negate ?? false, valueLabelConfig)}
-      iconOnClick={() => props.onRemove()}
-      onClick={handleBadgeClick}
-      data-test-subj={getDataTestSubj(valueLabelConfig)}
-    />
-  );
+  const filterViewProps = {
+    filter,
+    valueLabel: valueLabelConfig.title,
+    filterLabelStatus: valueLabelConfig.status,
+    errorMessage: valueLabelConfig.message,
+    className: getClasses(filter.meta.negate ?? false, valueLabelConfig),
```

Comment:
nit: why not `!filter.meta.negate`

Reply:
`!!filter.meta.negate` would be correct in this case.